rudder-lang/src/ir/ir1.rs

```diff
@@ -468,7 +468,7 @@ mod tests {
             "let c2 = case { X => \"val\", Y => \"val2\" }\n",
         );
 
-        parse_str_ok(&mut source, "ext1", "v=if X => \"val\"\n");
+        // parse_str_ok(&mut source, "ext1", "v=if X => \"val\"\n");
```

Comment:
why doesn't this work aymore ?

Reply:
forgot to uncomment it, fixed, and test passes